woocommerce/woocommerce-android
# Issue/5877 new inbox screen

Wip for: #5877


### Description
Displays a list of notes inside the new Inbox screen. The data is hard-coded at the moment. Just working on the UI side. 
This feature is hidden under a feature flag so it is safe to merge

### Testing instructions
Open Inbox and check design matches designs vbzgTKHV9XsyyN5cxxAuhv-fi-2669%3A91520

### Images/gif


| Dark  | Light |
| ------------- | ------------- |
| <img width="300"  src="https://user-images.githubusercontent.com/2663464/155006817-72358b0f-b60c-49c5-bf44-20300c083f97.png">  | <img width="300"  src="https://user-images.githubusercontent.com/2663464/155006708-f2edc2d4-ca81-4886-bf23-c63968debdbe.png">  |
| Content Cell  | Content Cell  |
